scripts/src/javatests/org/oppia/android/scripts/docs/KDocValidityCheckTest.kt

```diff
@@ -246,8 +383,84 @@ class KDocCheckTest {
     assertThat(exception).hasMessageThat().contains(KDOC_CHECK_FAILED_OUTPUT_INDICATOR)
     val failureMessage =
       """
-      ${retrieveTestFilesDirectoryPath()}/TempFile.kt:1: missing KDoc
-      ${retrieveTestFilesDirectoryPath()}/TempFile.kt:2: missing KDoc
+      KDoc missing for files:
+      - ${retrieveTestFilesDirectoryPath()}/TempFile.kt:1
+      - ${retrieveTestFilesDirectoryPath()}/TempFile.kt:2
+      """.trimIndent()
+    assertThat(outContent.toString().trim()).isEqualTo(failureMessage)
+  }
+
+  @Test
+  fun testKDoc_blockCommentInsteadOfKDoc_checkShouldFail() {
+    val testContent =
+      """
+      // Test block comment.
```

Comment:
This is a line comment, not a block comment (suggest updating the test name & adding a test for a block comment).

Reply:
Done.